README.md

```diff
@@ -41,6 +41,9 @@ If not running OS X, or you don't like Homebrew, you can use [pip](https://pip.p
 # Install Mackup with PIP
 pip install mackup
 
+# Update an existing version of Mackup with PIP
+pip install -upgrade mackup
```

Comment:
This would be `--upgrade`, but I suggest you specify that it's either `pip install` or `pip install --upgrade` and not both.